11ty/definitions/dissociative-identity-disorder.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Dissociative Identity Disorder
+slug: 'dissociative-identity-disorder'
+speech: noun
+defined: true
+---
+
+Dissociative identity disorder, also called DID, is a mental illness that involves disruptions or breakdowns of memory, awareness, identity and/or perception. It can also include the existence of two or more distinct identities (or “personality states”). The distinct identities are accompanied by changes in behavior, memory and thinking.
```

Comment:
```suggestion
Dissociative identity disorder (DID) is a mental illness that involves disruptions or breakdowns of memory, awareness, identity and/or perception. It can also include the existence of two or more distinct identities (or “personality states”). The distinct identities are accompanied by changes in behavior, memory and thinking.
```